modules/admin_manual/pages/maintenance/migrating_to_kiteworks.adoc

```diff
@@ -0,0 +1,398 @@
+
+== Prerequisites
+
+To be prepared for the migration, both sides need to match the prerequisites. Please read this section carefully.
+
+=== ownCloud
+
+* As a major prerequisite, the ownCloud instance *must* be running on release 10.14 or higher. If this requirement is not met, migration can not be started as the necessary app checks the minimum version.
+
+* Shell access is necessary. +
+`occ` commmands need to be started and `rclone` needs to be granted executable.
+
+* Install the ownCloud-provided migration components to the ownCloud instance, which are:
+** the migration app
+** the custom `rclone` binary 
+
+// install rclone from github.com/oks-maytech/rclone/tree/kiteworks-backend
+// install app from github.com/owncloud/migrate_to_kiteworks
+
+* All users must have an email address and they must be unique. +
+The `occ migration:verify` step will point out missing email addresses. These must be rectified before any migration can start.
+
+* ownCloud highly recommends installing and enabling, if not already present and enabled, the {oc-marketplace-url}/apps/impersonate[Impersonate] app. This app can be used to solve file and folder casing conflicts that can be reported during the migration process.
+
+==== Installing Required Components
+
+* Before you install the migration app, you should install the custom rclone binary first. The migration app can not be activated if rclone is missing. If you have installed rclone before, we recommend to uninstall and install the custom rclone binary instead. Post installing the rclone binary and making it execuatble, check the output of:
++
+--
+[source,bash]
+----
+rclone help backends
+----
+
+If the correct binary is installed and executed, it shows a list including: `Kiteworks`.
+
+[NOTE]
+====
+* You may need to specify the path to execute rclone if not part of the `$PATH` variable.
+* You can define the rclone binary used in the *occ migration command* by specifying the path along with other rclone relevant environment variables. In case, adapt the example command shown in the xref:migration[migration section] as follows:
++
+[source,bash]
+----
+sudo -u www-data \
+  PATH=/usr/local/bin:$PATH \
+  <other rclone envvars> \
+  php /var/www/owncloud/occ \
+  migrate:to-kiteworks \
+  ....
+----
+====
+
+rclone expects a config file and will report if it is missing which can pollute the migration log file. This config file only needs to be present but no content is necessary to avoid the message. If you see in the migration log the following message: `/<user-name>/.config/rclone/rclone.conf not found - using defaults`, you can create an empty `clone.conf` file with the following command. Note that you can do this before the migration starts. Replace `<user-name>` with the user you are logged in as.
```

Comment:
```suggestion
rclone expects a config file and will report if it is missing which can pollute the migration log file. An empty config file is sufficient to avoid the message. If you see in the migration log the following message: `/<user-home>/.config/rclone/rclone.conf not found - using defaults`, you can create an `rclone.conf` file with the following command. Replace `<user-home>` with the user you are logged in as.
```